DOCS - Reorganize With Diataxis (#377)
iotaledger/stronghold.rs: documentation/docs/how_tos/_README.mdx

---
description: How Tos give users step by step instructions on how to complete a concrete task.
image: /img/logo/iota_logo.svg
keywords:
  - How Tos
  - action oriented
  - how
---
import Tabs from "@theme/Tabs";
import TabItem from "@theme/TabItem";

# How To Files

> How Tos are analogous to [how-tos in Diataxis](https://diataxis.fr/explanation/). You should use this folder to give in depth guides on how to complete concrete single tasks. For example:

Reviewer: ```suggestion
How Tos are the same as [how-tos in Diataxis](https://diataxis.fr/how-to-guides/). You should use this folder to give in depth guides on how to complete concrete single tasks. For example:
```

- Connect to a Server
- Send a Message
- Create a Transaction
- Connect to Stronghold

> ## Action Oriented

Reviewer: ```suggestion
## Action-Oriented
```

> How To files should be **action oriented**. This means that you should first think of what you want the user to achieve

Reviewer: ```suggestion
How To files should be **action-oriented**. This means that you should first think of what you want the user to achieve
```
